files/image_config/constants/constants_use_neig.yml

```diff
@@ -0,0 +1,60 @@
+constants:
```

Comment:
@geogchen This UT looks good to me, but adding one new file under this folder maybe not suitable, we use one unified constants.yml for all platforms, could we mock the data in the testcases other than add a new file in this folder just for different parameter changes?
And there is an internal repo https://msazure.visualstudio.com/One/_git/Networking-acs-buildimage, I checked the constants.yml, which has use_neighbors_meta set to true, maybe you can just keep the UT part and when it synced into internal, you can run the UT with internal repo with the value "ture" for use_neighbors_meta.

Reply:
@geogchen Suggest to create a folder under src/sonic-bgpcfgd/tests/data, to hold variant constants.yaml files for test purpose.

Reply:
added a constants folder under tests/data, and moved the variant constants.yml to this new folder.